public/index.html

```diff
@@ -71,6 +71,14 @@
         <title>AntAlmanac</title>
         <!-- Global site tag (gtag.js) - Google Analytics -->
 
+        <!-- console.log for recruitment -->
+        <script>
+            console.log(
+                '%cInterested in improving AntAlmanac?\n - Checkout the project on GitHub: https://github.com/icssc-project/antalmanac\n - Join our Discord: https://discord.gg/GzF76D7UhY\n - Leave feedback: https://antalmanac.com/feedback',
+                'color: red; font-size: 25px; font-family: Comic Sans'
```

Comment:
Can you do the following?
- change the color to be the same color as AntAlmanac's navbar (you'll have to look it up in the code)
- remove the comic sans font family (it doesn't look like it renders anyways)

Reply:
![image](https://user-images.githubusercontent.com/74622613/158750130-25d367b2-d43b-4b23-a92f-e34aeeba0a6c.png)
like this?

Reply:
Looks great!